articles/anomaly-detection/references/attack-protection-triggers-actions.md

```diff
@@ -1,15 +1,17 @@
 ---
-title: Brute-Force Protection Triggers and Actions
-description: Brute-force protection triggers and actions taken upon anomaly detection and how blocks are cleared.
+title: Attack Protection Triggers and Actions
+description: Attack protection triggers and actions taken upon anomaly detection and how blocks are cleared.
 topics:
     - security
     - anomaly-detection
     - brute-force-protection
+    - credential-stuffing
+    - breached-password-protection
 contentType: reference
-useCase: customize-anomaly-detection
+useCase: anomaly-detection
 v2: true
 ---
-# Brute-Force Protection Triggers and Actions
+# Attack Protection Triggers and Actions
```

Comment:
Another thought: one question that came up during the meeting w/ security dashboard team was that they were uncertain which event types were generated by the various triggers.  Could be helpful to enumerate those here?

Reply:
Axe the whole doc - move anything that is only mentioned here to where it belongs.